x-pack/plugin/eql/qa/common/src/main/resources/test_queries_unsupported.toml

```diff
@@ -8,9 +8,178 @@
 # in order to allow at least one round-trip test with the test harness.
 # This will be removed once the EQL implementation is wired and actually supports this query.
 
-# [[queries]]
-# query = 'process where serial_event_id = 1'
-# expected_event_ids  = [1]
+
+
+##########################
+# FAILING SEQUENCE TESTS #
```

Comment:
@costin do you have any guesses why some of these sequence tests were failing?

Reply:
I believe it's because of the tie breaker - the logic only looks at the timestamp but since it's the same for all events, it fails to order it by the `seq_number`.

Reply:
Should be fixed now that the tiebreaker is in. It is disabled by default however our tests make use of it.